packages/protoplugin/src/ecmascript/generated-file.ts

```diff
@@ -66,7 +66,12 @@ export interface GeneratedFile {
    * - ImportSymbol: Adds an import statement and prints the name of the symbol.
    * - DescMessage or DescEnum: Imports the type if necessary, and prints the name.
    */
-  print(...any: Printable[]): void;
+  print(...printables: Printable[]): void;
+
+  /**
+   * Add a line of code to the file with tagged template literal.
```

Comment:
`Add a line of code to the file with tagged template literal and an optional array of Printables.  See print(Printable[]) for behavior when printing Printable items.`